### ISSUE TITLE
Feature Request | IP camera option on client

### ISSUE THREAD

**Issue by jedi404** [mfreiholz/ts3video#41]
there are several members of my community that would like to use the client with us but dont have traditional webcams. It would be a nice feature to be able to setup a small ip camera server to be able to start from the client if the option was selected that would allow users  to directly connect their phones camera to the client without 3rd party  software in the middle.

**Comment by 2E0PGS** [comment]
Awesome +1 this.

**Comment by mfreiholz** [comment]
Only for my understanding: You want to use an IP camera, which probably serves some kind of video via  ethernet (MPEG, MP4, ...) and use it as the client's camera source, right?

**Comment by jedi404** [comment]
yes, I have users without webcams and there exist a plethora of options to
turn the camera on your phone into an ip cmaera. having an option built in
to accept ip cameras as a source could streamline this, as well as allow
for users to set up dedicated feeds with traditional ip cameras if they
wanted. overall the feature would allow better compatibility and also the
ability to expand the functionality of the software.

**Comment by Bluscream** [comment]
You could use ManyCam to tunnel the output stream of your smartphone camera to the input stream of ManyCam's virtual webcam that you can use in TS3Video.
